danspeech/deepspeech/model.py

```diff
@@ -592,22 +535,142 @@ def forward(self, x, is_first, is_last):
         x = self.inference_softmax(x)
         return x
 
+
+    def get_seq_lens(self, input_length):
+        """
+        Given a 1D Tensor or Variable containing integer sequence lengths, return a 1D tensor or variable
+        containing the size sequences that will be output by the network.
+        :param input_length: 1D Tensor
+        :return: 1D Tensor scaled by model
+        """
+        seq_len = input_length
+        for m in self.conv.modules():
+            if type(m) == nn.modules.conv.Conv2d:
+                seq_len = ((seq_len + 2 * m.padding[1] - m.dilation[1] * (m.kernel_size[1] - 1) - 1) / m.stride[1] + 1)
+        return seq_len.int()
+
     @classmethod
     def load_model(cls, path):
+        """
+        If you do not need meta data from package, simply use this method to load the model.
+
+        :param path: Path to .pth package for the model.
+        :param conv_layers: Number of Conv layers. Will be removed after repackaging models.
```

Comment:
This isn't the case, is it? We need to differentiate between different number of convs?

Reply:
Fixed after repackageing. Forgot about comment! Nice spot. :)